geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseQueryPlan.scala

```diff
@@ -35,6 +35,7 @@ object HBaseQueryPlan {
     explainer.pushLevel(s"${prefix}Plan: ${plan.getClass.getName}")
     explainer(s"Table: ${Option(plan.table).orNull}")
     explainer(s"Ranges (${plan.ranges.size}): ${plan.ranges.take(5).map(rangeToString).mkString(", ")}")
+    explainer(s"Remote Filters: ${plan.remoteFilters.map(f => f.getClass.getCanonicalName).mkString(", ")}")
```

Comment:
yeah not very readable...wondering if we just use toString and override ours to output some config might work. The default tostring is the classname